strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/yaml/configuration/repository/MavenRepositoryConfigurationDto.java

```diff
@@ -22,6 +22,8 @@
     // defaults to once daily at 2 am
     private String cronExpression = "0 0 2 * * ?";
 
+    private String metadataExpirationStrategy = "checksum";
```

Comment:
The value here should be `MetadataExpirationStrategyType.CHECKSUM.name()` or `MetadataExpirationStrategyType.CHECKSUM.toString()` so that it continues to work if there is a future refactoring. :)

It looks like we missed this in the initial review, sorry.